packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -246,11 +247,11 @@ const start = async (zcf, privateArgs) => {
     const collateralKeyword = getKeywordForBrand(collateralAmount.brand);
     if (
       !AmountMath.isGTE(
-        collateralSeat.getCurrentAllocation()[collateralKeyword],
+        collateralSeat.getCurrentAllocation()[+collateralKeyword],
         collateralAmount,
       )
     ) {
-      throw new Error('insufficient reserves for that transaction');
+      throw assert.fail('insufficient reserves for that transaction');
```

Comment:
`throw` is redundant with `assert.fail`, yes? I think we have an `assert.error` constructor.

Reply:
Yes, the `throw assert.fail` should normally be just `assert.fail`. Because `assert.fail` @returns `{never}` the two should also be equivalent to typescript.

I have encountered rare circumstances where the `throw` was still necessary to inform some static check (probably es-lint) that control flow would not proceed normally past that point. Hope that you don't run into that.